test/bot.test.js

```diff
@@ -282,6 +282,13 @@ describe('Bot', function () {
     this.bot.parseText('<!somecommand> <!somecommand|readable>')
       .should.equal('<somecommand> <readable>');
   });
+  it('should handle entity-encoded messages from slack', function () {
```

Comment:
sure, i'll add newline. originally left that away to kinda tie that together with previous test.